src/components/menu/menu.css

```diff
@@ -62,4 +60,9 @@
         max-width: 100%;
         width: 400px;
     }
+
+    .menu-close-button {
```

Comment:
this one is a "total huita". You better do it with paddings

Reply:
let's discuss it on the "planerka"